src/ch/ch_monitor.c

```diff
@@ -1530,6 +1559,17 @@ virCHMonitorPingVMM(virCHMonitorPtr mon)
     return virCHMonitorGet(mon, URL_VMM_PING, NULL);
 }
 
+int
+virCHMonitorResizeCPU(virCHMonitorPtr mon,
+                      unsigned int nvcpus)
+{
+    g_autofree char *payload = NULL;
+    if (virCHMonitorBuildResizeJson(&nvcpus, &payload))
```

Comment:
Do we need to pass nvcpus as a pointer? I don't see the value being modified in any of the functions above in the stack.

Reply:
Ah thanks for pointing that out. I was considering something else alongside doing the memory hotplug and the pointer was a leftover from that design. Fixing.